server/delegator/delegator.route.js

```diff
@@ -16,4 +16,10 @@ router
 
   .get(validate(paramValidation.getByAddress), getDelegator)
 
+/** GET /api/delegators/lastRewards/:address - Get delegator last 30 rewards by address */
+router
+  .route('/lastRewards/:address')
```

Comment:
Req: The standard best practice for REST APIs is to have a hyphen, not camelcase or underscores. Also It is recommended to use hyphen (which is highlighted by [RFC3986](https://www.ietf.org/rfc/rfc3986.txt)).

Regards

Reply:
It's done